osu.Game/Overlays/Profile/UserGraph.cs

```diff
@@ -208,6 +208,7 @@ private Vector2 calculateBallPosition(int index)
 
         protected abstract class UserGraphTooltip : VisibilityContainer, ITooltip
         {
+            protected new readonly OsuSpriteText Name;
```

Comment:
Really don't like this being exposed as a `new`.. That's fine for tests but let's not do it in actual logic. Looks like you're only using the string portion of this anyway, so why not expose `tooltipCounterName` instead?

Reply:
It's not even being used anywhere as far as I can tell. Oversight maybe?

Reply:
Ah, is a drive-by change. Can be removed.